src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutManagerRecoverabilityPolicy.cs

```diff
@@ -2,25 +2,18 @@
 {
     using Transport;
 
-    class TimeoutManagerRecoverabilityPolicy : IRecoverabilityPolicy
+    class TimeoutManagerRecoverabilityPolicy
```

Comment:
I don't think we need the whole class here. Let's inline the method in `TimeoutManager.cs`